qa/tasks/salt_manager.py

```diff
@@ -123,18 +129,39 @@ def all_minions_zypper_status(self):
         self.all_minions_zypper_lu()
         self.all_minions_zypper_ps()
 
+    def cat_salt_master_conf(self):
+        self.__cat_file_remote(self.master_remote, filename="/etc/salt/master")
+
+    def cat_salt_minion_confs(self):
+        self.__cat_file_cluster(filename="/etc/salt/minion")
+
     def check_salt_daemons(self):
         self.master_remote.run(args=['sudo', 'salt-key', '-L'])
-        self.master_remote.run(args=[
-            'sudo', 'systemctl', 'status', 'salt-master.service'
-            ])
+        base_cmd = 'sudo systemctl status --full --lines={} {}.service'
```

Comment:
It is better to have embedded function for this purpose:
```
def check_service(self, name):
    def service_status_cmd(remote, service, lines):
         return 'sudo systemctl status --full --lines={} {}.service'.format(lines, service)
    try:
        remote.sh(service_status_cmd(name, 0))
    except CommandFailedError:
        remote.sh(service_status_cmd(name, 100))
self.check_service(self.master_remote, 'salt-master')
for r in self.ctx.cluster.remotes.iterkeys():
    self.check_service(r, 'salt-minion')
.
.
.

```

Reply:
cool. Added to wip-qa-cleanup.